.github/workflows/tests.yml

```
@@ -0,0 +1,42 @@
+name: Test
+
+on:
+  push:
+  # Triggers the workflow on labeled PRs only.
+  pull_request_target:
+    types: [labeled]
```

```suggestion
on:
  push:
    branches:
      - '**'
    tags-ignore:
      - '**'
  pull_request:
    types: [labeled]
```
`pull_request` is recommended, also we want this to run on every push on the default branch as well